xWorkflows/Xworkflow1/manifests/unclaimedvolumebackend/service.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: unclaimed-pv-api-service
+spec:
+  selector:
+    app: unclaimed-pv-api
+  ports:
+  - protocol: TCP
+    port: 5000
+    targetPort: 5000
+  type: NodePort
```

Comment:
why we are not using clusterIP

Reply:
For now, we need to test with the backend with Postman, that is why this is used.